hr_employee_passport/models/hr_employee_passport.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrEmployeePassport(models.Model):
+
+    _name = 'hr.employee.passport'
+    _inherit = 'mail.thread'
+    _description = 'Employee Passport'
+    _order = 'end_date desc'
+
+    name = fields.Char(
+        compute='_compute_name',
+        store=True,
+    )
+    description = fields.Char()
+    employee_id = fields.Many2one(
+        comodel_name='hr.employee',
+        string="Employee",
+        required=True,
+        ondelete='cascade',
+    )
+    end_date = fields.Date(
+        string="Expiration date",
+        required=True,
+        track_visibility='onchange',
+    )
+    passport_file = fields.Binary(
+        string="Passport",
+        attachment=True,
+        required=True,
```

Comment:
What about `required=False`?
Because I don't like my passport here

Reply:
The purpose of this module is store the passport, not only related data, but also a scan of the document.

Reply:
I agree it shouldn't be required by default.